src/plugins/orangehrmPerformancePlugin/Dao/PerformanceReviewDao.php

```diff
@@ -385,37 +385,44 @@ public function getReviewById(int $id): ?PerformanceReview
 
     /**
      * @param ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams
+     * @param string $reviewerGroupName
      * @return Kpi[]
      */
-    public function getKpisForReview(ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams): array
+    public function getKpisForReview(ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams, string $reviewerGroupName): array
     {
-        $qb = $this->getKpisForReviewQueryBuilderWrapper($reviewKpiSearchFilterParams)->getQueryBuilder();
+        $qb = $this->getKpisForReviewQueryBuilderWrapper($reviewKpiSearchFilterParams, $reviewerGroupName)->getQueryBuilder();
         return $qb->getQuery()->execute();
     }
 
     /**
      * @param ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams
+     * @param string $reviewerGroupName
      * @return int
      */
-    public function getKpisForReviewCount(ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams): int
+    public function getKpisForReviewCount(ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams, string $reviewerGroupName): int
```

Comment:
```suggestion
    public function getKpisForReviewCount(ReviewKpiSearchFilterParams $reviewKpiSearchFilterParams): int
```
better to move `$reviewerGroupName` into `ReviewKpiSearchFilterParams`

Reply:
Fixed but will push once overall testing on local runs and finish